acceptance-tests/coreclr.mk

```diff
@@ -4003,6 +4001,14 @@ CORECLR_DISABLED_TEST_CS_SRC += 	\
 # this test makes no sense, the expected pattern is the English one for all cultures
 CORECLR_DISABLED_TEST_CS_SRC += $(CORECLR_PATH)/tests/src/Regressions/coreclr/0584/test584.cs
 
+# Requires precise stack scanning
+CORECLR_DISABLED_TEST_CS_SRC +=        \
+       $(CORECLR_PATH)/tests/src/JIT/Directed/lifetime/lifetime2.cs
+
+# Depends on small array behavior of .net. Mono supports objects > 2Gb on 64bits and the following tests verify for that
+CORECLR_DISABLED_TEST_CS_SRC +=        \
+       $(CORECLR_PATH)/tests/src/JIT/Directed/newarr/newarr.cs
```

Comment:
[configure.ac#L2637](https://github.com/mono/mono/blob/d2cc9be75b10cd5d96a85dde6aff2ed014c1a6f7/configure.ac#L2637) says it defaults to no, and MONO_BIG_ARRAYS is not defined in config.h for me.